tests/TypedValueTests.cpp

```diff
@@ -69,3 +72,14 @@ TEST(TypedValue, typed_value_array_GetInts)
   ASSERT_EQ(1, values.at(1).int32_value());
   ASSERT_EQ(2, values.at(2).int32_value());
 }
+
+TEST(TypedValue, get_mixed_array_items)
+{
+  std::string filepath = TEST_MODEL_PATH  + "/issue-48.skp";
+  CW::initialize();
+  auto model = CW::Model(filepath);
+  CW::TypedValue typed_value_array = model.get_attribute("TypedValues", "Array<mixed>", typed_value_array);
```

Comment:
using `typed_value_array` as the 3rd parameter is probably incorrect here - should use a new `TypedValue` or maybe null as the default.

Reply:
Yea, that should be a default value. Doesn't make sense to pass in the variable that is receiving the results. (Surprised that compiles. I'm guessing `typed_value_array` is declared and initialized before `model.get_attribute` is called...)